codalab/rest/account.py

```diff
@@ -111,7 +112,7 @@ def do_signup():
 
     # Create unverified user
     _, verification_key = local.model.add_user(
-        username, email, first_name, last_name, password, affiliation
+        username, email, first_name, last_name, password, affiliation, 2
```

Comment:
Define `SEND_NOTIFICATIONS_ALL` instead of use bare constants....  (see how PERMISSIONS is defined)